File: csharp/examples/PostCallbacks.cs
using System;
namespace Examples.examples
{
    // Example of POST /callbacks
    
    public class PostCallbacks : BaseRequest
    {
        private string _callbackUrl;

        public PostCallbacks()
		{
            _callbackUrl = Environment.GetEnvironmentVariable("CALLBACK_URL");
            Console.WriteLine($"PostCallbacks: _callbackUrl={_callbackUrl}");
        }

        public override string SendRequest()
        {
            var body = Newtonsoft.Json.JsonConvert.SerializeObject(new
            {
                url = _callbackUrl
            });

            return PostRequest("callbacks", body);
        }
    }
}

File: php/examples/post_callbacks.php
<?php
require_once 'base_request.php';

// Example of POST /callbacks

$debug = true;
$url = $_ENV['CALLBACK_URL'];

$request_data = array(
    'url' => $url
);

$request_params = '';

$json_body = json_encode($request_data);

send_request('POST', '/callbacks', $request_params, $json_body);